src/Components/Background/index.ts

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+interface BackgroundProps {
+    imageUrl: string
+}
+
+const Background = styled("div")<BackgroundProps>`
```

Comment:
I am not a massive fan of `Background` name, `ImageHero` is much more appropriate, it tells the devs that's the image